src/components/common/ColorPicker/ColorPickerPopup.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import styles from './ColorPickerPopup.scss';
+import PropTypes from 'prop-types';
+import Button from '../Input/Button/Button';
+import ColorPicker from './ColorPicker';
+var { Checkboard } = require('react-color/lib/components/common');
+
+class ColorPickerPopup extends Component {
```

Comment:
This should be made into a generic popout component as well, so that it can be used in other scenarios